### PR TITLE
hid radio inputs, refactored functions and reset button

### PR DESCRIPTION
**Pull request by Thomas-Stussi**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -50,15 +58,15 @@ shootButton.addEventListener('click', () => {
     setTimeout(function() {
         resultSpan.classList.remove('hidden');
         if (matchResult === 'draw') {
-            resultSpan.textContent = 'We tied, lets go again';
+            resultSpan.textContent = `The computer played ${computerString}. We tied, lets go again`;
```
Comment (dpcairns) [inline app.js:61]: thoughtful addition

File: app.js [hunk 2]
```diff
@@ -26,15 +35,14 @@ let resetCounter = 0;
 // set event listeners to update state and DOM 
 
 startGame.addEventListener('click', () => {
-    startGame.classList.add('hidden');
-    theGame.classList.remove('hidden');
     computerDraw = getRandomThrow();
-    
+    toggleElement([startGame, theGame]);
```
Comment (dpcairns) [inline app.js:39]: woooah!! very cool!

File: gameUtils.js [hunk 3]
```diff
@@ -10,23 +10,27 @@ export function getRandomThrow() {
...
+    }
+    if (number === 2) {
+        return 'Paper';
+    }
+    if (number === 3) {
+        return 'Scissors';
+    }
+}
+
+export function toggleElement(elements){
```
Comment (dpcairns) [inline gameUtils.js:34]: this function is really cool! nice work. might want to add a check to make sure an array is passed in

File: app.js (hunk 2 above)
Comment (benwaples) [reply app.js:39]: Thank you! This ended up saving a good chunk of code I am stoked you noticed it :)

**Pull request merged**